### PR TITLE
Add test for addons named period, change message

### PR DESCRIPTION
**Pull request by johnotander**
Related #5487

### CODE REVIEW

File: tests/unit/commands/addon-test.js [hunk 1]
```diff
@@ -63,4 +63,13 @@ describe('addon command', function() {
       expect(error.message).to.equal('We currently do not support a name of `123-my-bagel`.');
     });
   });
+
+  it('doesn\'t allow to create an addon when the name is a period', function() {
+    return command.validateAndRun(['.']).then(function() {
+      expect(false, 'should have rejected with period as the addon name');
```
Comment (Turbo87) [inline tests/unit/commands/addon-test.js]: see #5483

**Comment by Turbo87** [comment]
LGTM. @kellyselden r?

**Comment by kellyselden** [comment]
@homu r+

**Comment by kellyselden** [comment]
Thank you!

**Pull request merged**